<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-03T15:33:14 IST" name="Suite_to_automate_SIM" finished-at="2024-07-03T15:33:39 IST" duration-ms="24966">
    <groups>
    </groups>
    <test started-at="2024-07-03T15:33:14 IST" name="Test" finished-at="2024-07-03T15:33:39 IST" duration-ms="24966">
      <class name="Sim_automation.InvoiceCreation">
        <test-method is-config="true" signature="reportGenerate()[pri:0, instance:Sim_automation.InvoiceCreation@cc6460c]" started-at="2024-07-03T15:33:14 IST" name="reportGenerate" finished-at="2024-07-03T15:33:14 IST" duration-ms="305" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportGenerate -->
        <test-method is-config="true" signature="browserOpen()[pri:0, instance:Sim_automation.InvoiceCreation@cc6460c]" started-at="2024-07-03T15:33:14 IST" name="browserOpen" finished-at="2024-07-03T15:33:20 IST" duration-ms="5380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserOpen -->
        <test-method signature="invoiceform()[pri:0, instance:Sim_automation.InvoiceCreation@cc6460c]" started-at="2024-07-03T15:33:20 IST" name="invoiceform" description="Add an Invoice in the SIM-Web App" finished-at="2024-07-03T15:33:38 IST" duration-ms="18322" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Customer Added successfully expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Customer Added successfully expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at Sim_automation.InvoiceCreation.invoiceform(InvoiceCreation.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invoiceform -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:Sim_automation.InvoiceCreation@cc6460c]" started-at="2024-07-03T15:33:38 IST" name="closeBrowser" finished-at="2024-07-03T15:33:39 IST" duration-ms="1089" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:Sim_automation.InvoiceCreation@cc6460c]" started-at="2024-07-03T15:33:39 IST" name="endReport" finished-at="2024-07-03T15:33:40 IST" duration-ms="378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- Sim_automation.InvoiceCreation -->
    </test> <!-- Test -->
  </suite> <!-- Suite_to_automate_SIM -->
</testng-results>
